marine/marine.py

```diff
@@ -50,12 +49,44 @@ def epan_auto_reset_count(self) -> int:
     def epan_auto_reset_count(self, value: int) -> None:
         self._marine.set_epan_auto_reset_count(value)
 
+    def filter(
+        self,
+        packet: bytes,
+        bpf: Optional[str] = None,
+        display_filter: Optional[str] = None,
+        encapsulation_type: int = encap_consts.ENCAP_ETHERNET,
+    ) -> bool:
+        passed, _ = self.filter_and_parse(
+            packet=packet,
+            bpf=bpf,
+            display_filter=display_filter,
+            encapsulation_type=encapsulation_type,
+        )
+
+        return passed
+
+    def parse(
+        self,
+        packet: bytes,
+        fields: Optional[List[str]] = None,
+        encapsulation_type: int = encap_consts.ENCAP_ETHERNET,
+        macros: Optional[Dict[str, List[str]]] = None,
+    ) -> Dict[str, str]:
```

Comment:
`Dict[str, Optional[str]]`

Reply:
Those are changes that are related to the [empty-fields-type](https://github.com/tomlegkov/marine-python/pull/33) PR so it would be nice if they were noted there as I indeed forgot to switch the typing :)